packages/itwinui-css/src/tabs/tabs.scss

```
@@ -99,6 +99,14 @@ $borderless-horizontal-tab-min-height: calc(
     }
   }
 
+  &.iui-overflow {
```

Can we use a data attribute instead?  Next major release I'd like to replace most, if not all, class modifiers with data attributes.
```suggestion
  &[data-iui-overflow='true'] {
```

---

Updated